hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/VolumeInfoMetrics.java

```diff
@@ -33,6 +35,7 @@ public class VolumeInfoMetrics {
 
   private String metricsSourceName = VolumeInfoMetrics.class.getSimpleName();
   private final HddsVolume volume;
+  private final AtomicLong dbCompactTimes = new AtomicLong();
```

Comment:
I believe the unit should be nano seconds? 
```suggestion
  private final AtomicLong dbCompactTimesNanoSeconds = new AtomicLong();
```

Reply:
sure.